website/app/controllers/application.js

```diff
@@ -28,17 +29,52 @@ export default class ApplicationController extends Controller {
     return this.router.currentURL.split('/')[1];
   }
 
+  resetSidebar() {
+    this.isSidebarVisibleOnSmallViewport = false;
+    if (!this.fastboot.isFastBoot) {
+      document.body.classList.remove('isSidebarVisibleOnSmallViewport');
+      document.body.classList.remove('isSidebarInRenderTree');
+      if (this.runNext) {
+        cancel(this.runNext);
+      }
+      if (this.runLater) {
+        cancel(this.runLater);
+      }
+    }
+  }
+
+  showSidebarOnSmallViewport() {
+    if (!this.fastboot.isFastBoot) {
+      document.body.classList.add('isSidebarInRenderTree');
+      if (this.runLater) {
+        cancel(this.runLater);
+      }
+      this.runNext = next(() => {
+        document.body.classList.add('isSidebarVisibleOnSmallViewport');
```

Comment:
Super nit: To me, the class name seems a little hard to understand. Would it make sense to use the wording from the function name instead?

IE, use: `document.body.classList.add('showSidebarOnSmallViewport');`